client/src/components/AttractionShowPage.js

```diff
@@ -31,12 +32,42 @@ const AttractionShowPage = (props) => {
     setAttraction({ ...attraction, reviews: [...attraction.reviews, review] });
   };
 
+  const deleteReview = async (reviewId) => {
+    try {
+      const response = await fetch(`/api/v1/reviews/${reviewId}`, {
+        method: "DELETE",
+        headers: { "Content-Type": "application/json" },
+      })
+      if (!response.ok) {
+        if (response.status === 422) {
+          const body = await response.json()
+          const newErrors = translateServerErrors(body.errors)
+          return setErrors(newErrors)
+        } else {
+          const errorMessage = `${response.status} (${response.statusText})`
+          const error = new Error(errorMessage)
+          throw error
+        }
+      } else {
```

Comment:
you control for 422, but your router isnt setup to respond with a 422 here. Instead, please prepare for a 401 error